lib/sord/generator.rb

```diff
@@ -614,11 +614,11 @@ def sort_params(pair1, pair2)
     # '-', working around a bug in YARD. (See lsegal/yard #894)
     #
     # @param [String] default
-    # @return [String]
+    # @return [String, nil]
     def fix_default_if_unary_minus(default)
       if default.nil?
         nil
-      elsif default[0] == '-'
+      elsif default[0] == '-' && !default.include?('->')
```

Comment:
It may be better to check `start_with?('->')` rather than `include?('->')`.

`include?` would skip the fix if a default value includes _both_ `->` and a unary minus, which is admittedly ridiculous yet valid, like this:

```ruby
def x(a: -(->{2}.())); end
```